sdk/nodejs/src/bucketing.ts

```diff
@@ -3,22 +3,25 @@ import { DVCLogger, DVCReporter } from '@devcycle/types'
 import { DevCycleOptions } from './types'
 
 let Bucketing: Exports | null
-let InstantiatePromise: Promise<Exports> | null
+let bucketingInstantiationCount = 0
+const bucketingInstantiationWarning = 3
 
 export const importBucketingLib = async ({
     logger,
     options,
 }: { logger?: DVCLogger; options?: DevCycleOptions } = {}): Promise<void> => {
-    if (InstantiatePromise) {
-        await InstantiatePromise
-        return
-    }
     const debugWASM = process.env.DEVCYCLE_DEBUG_WASM === '1'
-    InstantiatePromise = instantiate(debugWASM).then((exports) => {
+    await instantiate(debugWASM).then((exports) => {
```

Comment:
Can we add tests to make sure things work with multiple WASM libs? Probably also makes sense to add a test harness tests to make sure multiple clients work.

Reply:
yea for sure I much prefer reviewing a draft PR, really helps clarify what this change would mean. But yea in general I think this is a good idea.